docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -203,6 +203,9 @@ export default class MethodPage extends React.PureComponent<Props, State> {
                   onChange={this.onDebugFormChange}
                   inputProps={{
                     className: 'code',
+                    style: {
+                      fontSize: '14px',
```

Comment:
Does it work to set this in the stylesheet instead of using inline styles?

Reply:
This works in my local env. 
I removed it eventually, because everyone has a different taste in design. 
Maybe only me who thinks the `font-size` of `debug` and `response` may look too big.